op-bindings/predeploys/dev_addresses.go

```diff
@@ -10,6 +10,7 @@ const (
 	DevOptimismMintableERC20Factory = "0x6900000000000000000000000000000000000004"
 	DevAddressManager               = "0x6900000000000000000000000000000000000005"
 	DevProxyAdmin                   = "0x6900000000000000000000000000000000000006"
+	DevL1ERC721Bridge               = "0x6900000000000000000000000000000000000007"
```

Comment:
Why do we use these dev addresses?

Reply:
These dev addresses are used for deploying the L1 contracts to know addresses when building the L1 developer genesis